src/components/Card/Card.js

```diff
@@ -4,13 +4,13 @@ const Card = ({ name, type, id, maker, cost, complications, features, size  }) =
 
   return (
     <div className='card'>
-      <p>{name}</p>
-      <p>{type}</p>
-      <p>{maker}</p>
-      <p>{cost}</p>
-      <p>{complications}</p>
-      <p>{features}</p>
-      <p>{size}</p>
+      <p>Name: {name}</p>
```

Comment:
Nice to have additional information before your input your card information, lets the user bet a better idea of what they are looking at